code/modules/clothing/spacesuits/rig/modules/ds13/healthbar.dm

```diff
@@ -36,7 +36,8 @@
 		tracking_level = pick(RIG_SENSOR_OFF, RIG_SENSOR_BINARY, RIG_SENSOR_VITAL, RIG_SENSOR_TRACKING)
 
 /obj/item/rig_module/healthbar/Destroy()
-	unregister_user()
```

Comment:
doesnt that proc already do safety checks?

Reply:
Nope, but I think it'll be better to move this check to unregister_user()